### PR TITLE
FileManager: update validation section

### PR DESCRIPTION
**Pull request by AnnaKondratova**

### CODE REVIEW

File: concepts/05 Widgets/FileManager/30 Validation/20 Upload Settings.md [hunk 1]
```diff
@@ -0,0 +1,130 @@
...
+
+    <!-- tab: App.vue -->
+    <template>
+        <DxFileManager>   
+            <DxUpload :max-file-size="1000000" :chunk-size="500000" />
+            <DxPermissions :upload="true" />
+        </DxFileManager>
+    </template>
+    <script>
+        import 'devextreme/dist/css/dx.common.css';
```
Comment (RomanTsukanov) [inline concepts/05 Widgets/FileManager/30 Validation/20 Upload Settings.md]: We don't usually shift the contents of `<script>`.

File: concepts/05 Widgets/FileManager/30 Validation/20 Upload Settings.md [hunk 2]
```diff
@@ -0,0 +1,130 @@
+The widget allows you to configure upload settings: 
+
+- [maxFileSize](/api-reference/10%20UI%20Widgets/dxFileManager/1%20Configuration/upload/maxFileSize.md '/Documentation/ApiReference/UI_Widgets/dxFileManager/Configuration/upload/#maxFileSize') option - Specifies the maximum upload file size. The widget does not upload a file and displays the following error message when the file's size exceeds the maximum upload file size.
```
Comment (RomanTsukanov) [inline concepts/05 Widgets/FileManager/30 Validation/20 Upload Settings.md]: With this formating, all the text will be displayed on a single line. I suggest formatting the list as shown, for example, in [this file](https://raw.githubusercontent.com/DevExpress/devextreme-documentation/20_1/api-reference/10%20UI%20Widgets/GridBase/1%20Configuration/sorting/mode.md).

Comment (AnnaKondratova) [reply concepts/05 Widgets/FileManager/30 Validation/20 Upload Settings.md]: ok

File: concepts/05 Widgets/FileManager/30 Validation/20 Upload Settings.md (hunk 1 above)
Comment (AnnaKondratova) [reply concepts/05 Widgets/FileManager/30 Validation/20 Upload Settings.md]: updated

File: concepts/05 Widgets/FileManager/30 Validation/20 Upload Settings.md [hunk 3]
```diff
@@ -0,0 +1,132 @@
+The widget allows you to configure upload settings: 
+
+- [maxFileSize](/api-reference/10%20UI%20Widgets/dxFileManager/1%20Configuration/upload/maxFileSize.md '/Documentation/ApiReference/UI_Widgets/dxFileManager/Configuration/upload/#maxFileSize')
+Specifies the maximum upload file size. The widget does not upload a file and displays the following error message when the file's size exceeds the maximum upload file size.
```
Comment (RomanTsukanov) [inline concepts/05 Widgets/FileManager/30 Validation/20 Upload Settings.md]:
```suggestion
    Specifies the maximum upload file size. The widget does not upload a file and displays the following error message when the file's size exceeds the maximum upload file size.
```

File: concepts/05 Widgets/FileManager/30 Validation/20 Upload Settings.md [hunk 4]
```diff
@@ -0,0 +1,132 @@
...
+
+    ![DevExtreme File Manager - Max File Size](/images/FileManager/max-file-size-error-message.png)
+
+- [chunkSize](/api-reference/10%20UI%20Widgets/dxFileManager/1%20Configuration/upload/chunkSize.md '/Documentation/ApiReference/UI_Widgets/dxFileManager/Configuration/upload/#chunkSize')
+Specifies a chunk size to divide a large file in parts and upload them in separate requests.
```
Comment (RomanTsukanov) [inline concepts/05 Widgets/FileManager/30 Validation/20 Upload Settings.md]:
```suggestion
    Specifies a chunk size to divide a large file in parts and upload them in separate requests.
```

**Review by RomanTsukanov — APPROVED** [review]

**Pull request merged**